File: easy/nim_game/nim_game.py
class Solution(object):
    def canWinNim(self, n):
        """
        :type n: int
        :rtype: bool
        """
        return n % 4 != 0


s = Solution()
print("Solution 1 : ", s.canWinNim(4))
print("Solution 2 : ", s.canWinNim(5))
print("Solution 3 : ", s.canWinNim(8))
print("Solution 4 : ", s.canWinNim(16))

File: easy/nim_game/nim_game.go
package main

import "fmt"

func canWinNim(n int) bool {
	return n%4 != 0
}

func main() {
	fmt.Println("Solution 1 : ", canWinNim(4))
	fmt.Println("Solution 2 : ", canWinNim(5))
	fmt.Println("Solution 3 : ", canWinNim(8))
	fmt.Println("Solution 3 : ", canWinNim(16))
}
